src/common-principles.md

```diff
@@ -422,6 +422,10 @@ possible. Since the NIfTI standard offers limited support for the various image
 acquisition parameters available in DICOM files, we RECOMMEND that users provide
 additional meta information extracted from DICOM files in a sidecar JSON file
 (with the same filename as the `.nii[.gz]` file, but with a `.json` extension).
+Many of such metadata entries are explicitly formalized in BIDS and can be found listed in the [Glossary](./glossary.md).
+Whenever formalizing a new metadata entry within BIDS, the name SHOULD correspond to the original DICOM Tag whenever the value is taken *as is* without any further harmonization.
+The description of the term in the schema then SHOULD describe such correspondence as "Corresponds to DICOM Tag ID1, ID2 `DICOM Tag Name`.".
+If value is harmonized, the description SHOULD describe such correspondence as "Based on DICOM Tag ID1, ID2 `DICOM Tag Name`.".
```

Comment:
With just a slight tune up a [suggestion from @effigies](https://github.com/bids-standard/bids-specification/pull/1450#issuecomment-2076255999):

```suggestion
Currently defined metadata fields are listed in the [Glossary](https://github.com/bids-standard/bids-specification/pull/glossary.md).
Where possible, DICOM Tags are adopted directly as BIDS metadata terms and indicated with "Corresponds to DICOM Tag ID1, ID2 DICOM Tag Name.".
When harmonization has been deemed necessary, this is indicated in the BIDS term description with "Based on DICOM Tag ID1, ID2 DICOM Tag Name.".
```

I like it . WDYT @sappelhoff

Reply:
LGTM, but I think you forgot the backticks around `DICOM Tag Name`!

Reply:
With updated glossary URL, bold markers and back-ticks:

```suggestion
Currently defined metadata fields are listed in the [Glossary](./glossary.md).
Where possible, DICOM Tags are adopted directly as BIDS metadata terms and
indicated with "**Corresponds to** DICOM Tag ID1, ID2 `DICOM Tag Name`.".
When harmonization has been deemed necessary, this is indicated in the
BIDS term description with "**Based on** DICOM Tag ID1, ID2 `DICOM Tag Name`.".
```